servicetalk-opentracing-zipkin-publisher/src/test/java/io/servicetalk/opentracing/zipkin/publisher/reporter/HttpReporterTest.java

```diff
@@ -170,7 +171,10 @@ private void verifySpanSendFailuresAreRecovered(final Codec codec, final HttpRep
         responseGenerator = (httpServiceContext, factory) -> {
             try {
                 httpServiceContext.closeAsync().toFuture().get();
-            } catch (InterruptedException | ExecutionException e) {
+            } catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
+                throwException(e);
+            } catch (ExecutionException e) {
                 throw new RuntimeException(e);
```

Comment:
In all other places you rethrow `ExecutionException` using `throwException`. Can we be consistent here?